src/main/java/teammates/ui/pagedata/InstructorFeedbackEditPageData.java

```diff
@@ -38,16 +39,18 @@
     private CourseDetailsBundle courseDetails;
     private int numOfInstructors;
     private FeedbackSessionAttributes feedbackSession;
+    private List<FeedbackTemplateQuestionInfo> templateQnForm;
     private Map<String, String> resolvedTimeFields = new HashMap<>();
 
     public InstructorFeedbackEditPageData(AccountAttributes account, String sessionToken) {
         super(account, sessionToken);
     }
 
     public void init(FeedbackSessionAttributes feedbackSession, List<FeedbackQuestionAttributes> questions,
```

Comment:
>  Add Javadoc to state what is templateQuestions. For other parameters, you can add their description if you want.

Address this :P Its necessary to explain `templateQuestions`

Reply:
Sorry I missed that out previously! :(